homeassistant/components/media_player/squeezebox.py

```diff
@@ -203,9 +191,30 @@ def state(self):
                 return STATE_IDLE
         return STATE_UNKNOWN
 
-    def update(self):
-        """Retrieve latest state."""
-        self._status = self._lms.get_player_status(self._id)
+    @asyncio.coroutine
+    def async_query(self, *parameters):
```

Comment:
Change this function to the format here: https://github.com/home-assistant/home-assistant/pull/5001

It should remove the coroutine decorator and just return the coroutine generated by async_query.